.github/workflows/build.yml

```diff
@@ -3,11 +3,8 @@ name: build
 on:
   pull_request: ~
   push:
-    branches:
-      - master
-  release:
-    types:
-      - created
+    branches: [master]
+    tags: ['v*']
```

Comment:
This is risky. Since we introduce manual process to make a real release from the draft, I'm fine for now.

Reply:
discussed.